packages/block-library/src/image/edit.native.js

```diff
@@ -713,9 +759,15 @@ export class ImageEdit extends Component {
 		if ( ! url ) {
 			return (
 				<View style={ styles.content }>
+					{ isFetchingImage && (
+						<View style={ styles.image__loading }>
+							<ActivityIndicator animating />
+						</View>
+					) }
```

Comment:
Nitpick: I see this block is repeated in two places, I'm wondering if we could abstract it so we only need to have it in one place, wdyt?